homework/Vasili_Zasinets/Shape/src/main/java/com/tutrit/java/quickstart/shape/AbstractShape.java

```diff
@@ -0,0 +1,24 @@
+package com.tutrit.java.quickstart.shape;
+
+public abstract class AbstractShape implements Shape{
+    protected String name;
+
+
+    @Override
+    public double getArea() throws ShapeException {
+        throw new ShapeException("Incorrect in data entered");
```

Comment:
I don't see any data in method parameters, don't understand why you throwing ("Incorrect in data entered")

Reply:
ok. Fixed it